test/jarque_bera.jl

```diff
@@ -0,0 +1,58 @@
+using HypothesisTests, Base.Test
```

Comment:
Sorry, that was my mistake. I have now added all the missing tests to the file. Tests still pass :-)